### PR TITLE
Rnn comments only

### PR DESCRIPTION
**Pull request by peterdudfield**
Add comments into code

This are copied from the RNN branch

### CODE REVIEW

File: perceiver_pytorch/layers.py [hunk 1]
```diff
@@ -76,6 +88,15 @@ class Attention(nn.Module):
     def __init__(
         self, query_dim, context_dim=None, heads=8, dim_head=64, dropout=0.0
     ):
+        """
+
+        :param query_dim: Size of the queries.
```
Comment (jacobbieker) [inline perceiver_pytorch/layers.py:93]: Maybe redo in Google Style?

**Review by jacobbieker — APPROVED** [review]
LGTM! I think just one docstring slipped through not in Google style

File: perceiver_pytorch/layers.py (hunk 1 above)
Comment (peterdudfield) [reply perceiver_pytorch/layers.py]: thanks

File: perceiver_pytorch/layers.py [hunk 2]
```diff
@@ -91,12 +113,27 @@ def __init__(
...
+        """
+
         h = self.heads
 
         q = self.to_q(x)
         context = default(context, x)
         k, v = self.to_kv(context).chunk(2, dim=-1)
 
+        # Rearrange the query, key and value tensors.
+        # b = batch size; n =
```
Comment (JackKelly) [inline perceiver_pytorch/layers.py:135]: @jacobbieker do you know what `n` is here?  Size of the latent array?

File: perceiver_pytorch/perceiver_pytorch.py [hunk 3]
```diff
@@ -44,7 +43,9 @@ def __init__(
           depth: Depth of net.
           max_freq: Maximum frequency, hyperparameter depending on how
               fine the data is.
-          freq_base: Base for the frequency
+          freq_base: Base of the logarithm function for Fourier position
+              encoding.
+
```
Comment (JackKelly) [inline perceiver_pytorch/perceiver_pytorch.py:48]: Maybe delete this blank line?

**Review by JackKelly — APPROVED** [review]
LGTM!  Just 3 tiny comments!

**Pull request merged**